portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,6 +62,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String text = getParameter(request, "comment", "");
+    String quantity = getParameter(request,"quantity","4");
+    numOfComments = Integer.parseInt(quantity.equals("")?"1":quantity);
```

Comment:
This is saving the value from a post of a form, but this value will be quickly overwritten by the next post from another user.

Reply:
Yes I don't need to save the value. numOfComments is now a local variable. 9ecf11367307858c121189ed146b2a1c3ac88af3